07deBERTaClassifierForNewDataset.ipynb

```diff
@@ -0,0 +1,283 @@
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "folder_to_output = \"./outputs\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_model(params, db_config_base, model_nm):\n",
+    "    db_config = db_config_base\n",
+    "    if params is not None:\n",
+    "        db_config.update({\"cls_dropout\": params[\"cls_dropout\"]})\n",
+    "    db_config.update({\"num_labels\": 2})\n",
+    "    model = transformers.AutoModelForSequenceClassification.from_pretrained(model_nm, config=db_config)\n",
+    "    return model\n",
+    "\n",
+    "\n",
+    "def compute_metrics(eval_pred):\n",
+    "    predictions, labels = eval_pred\n",
+    "    predictions = predictions.argmax(axis=-1)\n",
+    "    classification_report = sklearn.metrics.classification_report(\n",
+    "        labels, predictions, target_names=[\"not relevance\", \"relevance\"], output_dict=True\n",
+    "    )\n",
+    "    f1_not_relevance = classification_report[\"not relevance\"][\"f1-score\"]\n",
+    "    f1_relevance = classification_report[\"relevance\"][\"f1-score\"]\n",
+    "    return {\"f1_not_relevance\": f1_not_relevance, \"f1_relevance\": f1_relevance}\n",
+    "\n",
+    "\n",
+    "def get_trainer(dataset, db_config_base, model_nm, folder_to_output, parameters):\n",
+    "    args = transformers.TrainingArguments(\n",
+    "        folder_to_output,\n",
+    "        learning_rate=parameters[\"learning_rate\"],\n",
+    "        warmup_ratio=parameters[\"warmup_ratio\"],\n",
+    "        lr_scheduler_type=parameters[\"lr_scheduler_type\"],\n",
+    "        disable_tqdm=False,\n",
+    "        fp16=True,\n",
+    "        evaluation_strategy=\"epoch\",\n",
+    "        per_device_train_batch_size=parameters[\"batch_size\"],\n",
+    "        per_device_eval_batch_size=parameters[\"batch_size\"],\n",
+    "        num_train_epochs=parameters[\"epochs\"],\n",
+    "        weight_decay=parameters[\"weight_decay\"],\n",
+    "        report_to=\"none\",\n",
+    "        save_strategy=\"epoch\",\n",
+    "        load_best_model_at_end=True,\n",
+    "    )\n",
+    "    get_model_partial = functools.partial(get_model, db_config_base=db_config_base, model_nm=model_nm)\n",
+    "    return transformers.Trainer(\n",
+    "        model_init=get_model_partial,\n",
+    "        args=args,\n",
+    "        train_dataset=dataset[\"train\"],\n",
+    "        eval_dataset=dataset[\"test\"],\n",
+    "        tokenizer=tokenizer,\n",
+    "        compute_metrics=compute_metrics,\n",
+    "    )\n",
+    "\n",
+    "\n",
+    "parameters = {\n",
+    "    \"learning_rate\": 8e-5,\n",
+    "    \"batch_size\": 16,\n",
+    "    \"weight_decay\": 0.01,\n",
+    "    \"epochs\": 1,\n",
+    "    \"warmup_ratio\": 0.1,\n",
+    "    \"cls_dropout\": 0.3,\n",
+    "    \"lr_scheduler_type\": \"cosine\",\n",
+    "}\n",
```

Comment:
did you use the baseline parameters that are used for training the rain classifier? if so, please add a comment clarifying this in the code